src/ui/configuration/CopterPidConfig.cc

```diff
@@ -155,119 +153,41 @@ CopterPidConfig::CopterPidConfig(QWidget *parent) : AP2ConfigWidget(parent)
     initConnections();
 }
 
-void CopterPidConfig::mapParamNamesToBox() {
-
-    // AC3.4+ param name compatibility
+void CopterPidConfig::mapParamNamesToBox()
+{
+    // version check
     ArduPilotMegaMAV* apmMav = static_cast<ArduPilotMegaMAV*>(m_uas);
```

Comment:
should be dynamic cast

Reply:
how to get these fixed code to be loaded in OSX? thx

Reply:
check this [](http://firmware.ardupilot.org/Tools/APMPlanner/daily/2019-03-08/) for the hot stuff
and this [](http://firmware.ardupilot.org/Tools/APMPlanner/) for the officials